nemesis/arena_battles.py

```diff
@@ -207,15 +207,20 @@ def yojimbo_battle():
     screen.await_turn()
     if not Yuna.active():
         battle.main.buddy_swap(Yuna)
-    logger.debug("Yuna Overdrive to summon Yojimbo")
-    battle.overdrive.yuna()
-    # needed_amount = min(round(zanmato_gil_needed(), -1) + 30, 263000)
-    needed_amount = 263000
-    logger.debug(f"Pay the man: {needed_amount}")
-    # battle.overdrive.yojimbo(gil_value=needed_amount) # still testing
-    battle.overdrive.yojimbo()  # Backup plan
-
-    memory.main.wait_frames(90)
+    if not Yuna.is_turn():
+        while memory.main.battle_active() and not Yuna.is_turn():
+            if memory.main.turn_ready():
+                CurrentPlayer().defend()
+    if memory.main.battle_active():
+        logger.debug("Yuna Overdrive to summon Yojimbo")
+        Yuna.overdrive(aeon_num=5)
```

Comment:
You should be able to pass in `Yojimbo` as the argument and it should just work.

Reply:
Will add this as a future enhancement.